### PR TITLE
redux-logger definitions

### PR DESCRIPTION
**Pull request by arusakov**
https://github.com/fcomb/redux-logger

### CODE REVIEW

**Comment by arusakov** [comment]
@theaqua, Can you review PR?

File: redux-logger/redux-logger.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+declare module 'redux-logger' {
+
+  interface ReduxLoggerOptions {
+    collapsed?: boolean;
+    level?: string;
+    logger?: any;
+    timestamp?: boolean;
+    transformer?: (state:any)=>any;
+    predicate?: (getState:Function, action:any)=>any;
```
Comment (theaqua) [inline redux-logger/redux-logger.d.ts]: `=> bool`, predicate must return true or false.

**Comment by arusakov** [comment]
@theaqua Thank you!
@vvakame Probably it's ready to merge?

**Comment by vvakame** [comment]
*redux-logger/redux-logger.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/redux-logger
  * http://bower.io/search/?q=redux-logger
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (redux-logger/redux-logger-tests.ts or others)
* [ ] pass the Travis-CI test?

**Comment by vvakame** [comment]
@theaqua thanks!
@arusakov thanks mate :+1:

**Pull request merged**